packages/remote-feature-flag-controller/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0]
+
+### Added
+
+- Initial release
```

Comment:
It's best if every changelog entry has a link to the PR that introduced this change. In this case, it's this very PR that introduces the package. So, how about:

``` suggestion
- Initial release ([#4931](https://github.com/MetaMask/core/pull/4931))
```